vck-openid/src/commonMain/kotlin/at/asitplus/wallet/lib/openid/RequestOptions.kt

```diff
@@ -121,6 +121,12 @@ data class RequestOptions(
 /**
  * Miscellaneous helper functions regarding [ConstraintField]
  */
+private fun Collection<ConstraintField>.toRequestedAttributes(): List<String> {
+    val regex = "[a-zA-Z0-9_-]+".toRegex()
+    val rawAttributes = this.map { constraint -> constraint.path.last().split("[").last() }
```

Comment:
This seems it should be handled by JSON Path? / cc @acrusage-iaik

Reply:
since we cannot use schema-matching according to [v2.0.0 or later](https://identity.foundation/presentation-exchange/spec/v2.1.1/#input-descriptor) this needs to be reworked.